python/sglang/srt/layers/attention/hybrid_linear_attn_backend.py

```diff
@@ -666,17 +694,31 @@ def forward_extend(
             )
             mixed_qkv = mixed_qkv_processed.transpose(1, 2).view(seq_len, -1)
         else:
-            mixed_qkv = causal_conv1d_fn(
-                mixed_qkv.transpose(0, 1),
-                conv_weights,
-                bias,
-                activation=activation,
-                conv_states=conv_states,
-                has_initial_state=has_initial_states,
-                cache_indices=cache_indices,
-                query_start_loc=query_start_loc,
-                seq_lens_cpu=forward_batch.extend_seq_lens_cpu,
-            ).transpose(0, 1)[:seq_len]
+            if _use_cpu:
```

Comment:
same as above.

i think it would be better to wrap torch.ops.sgl_kernel.causal_conv1d_fwd_cpu in a python file and get rid of the if here.